api_app/analyzers_manager/models.py

```diff
@@ -0,0 +1,30 @@
+from django.db import models
+from django.contrib.postgres import fields as postgres_fields
+
+
+class AnalyzerReport(models.Model):
+    STATUS_CHOICES = (
```

Comment:
Define an `Enum` class for Status. Like:

```python3
class Statuses(Enum):
     FAILED = 0
     PENDING = 1
     RUNNING = 2
     SUCCESS = 3
```

and then u may use these values like `Statuses.RUNNING.name` etc.

Reply:
Clarification:

The `Statuses` class will be defined outside `AnalyzerReport` but a reference of it will exist inside `AnalyzerReport`. 

```python3
from enum import Enum

class Statuses(Enum):
     FAILED = 0
     PENDING = 1
     RUNNING = 2
     SUCCESS = 3

class AnalyzerReport(models.Model):
    Statuses = Statuses
```

This is so we do not have to import `Statuses` everywhere instead we could simply do `AnalyzerReport.Statuses.RUNNING.name` or `self.report.Statuses.RUNNING.name` inside analyzer scripts.